requirements/min-reqs.old

```diff
@@ -12,7 +12,7 @@ numpy==1.23.2
 pandas==1.5.0
 pillow==9.2.0
 pyproj==3.4.0
-rasterio==1.3.3
+rasterio==1.3.11
```

Comment:
Somehow this slipped under my radar. Was there a reason we needed to change the minimum support rasterio version? We forgot to change this in `pyproject.toml`, which is what pip actually uses.

Reply:
Fixed in #2969